### ISSUE TITLE
Unable to call fake data csv when located in another folder

### ISSUE THREAD

**Issue by q-schen** [CityofToronto/bdit_king_pilot_dashboard#36]
Trying to load external data only works when the data file is in the same folder as the calling file. My current workaround is to copy-paste the data file into my volume map folder - this way I can move on. 

Problem and solution explained [here](https://stackoverflow.com/questions/41588875/d3-csv-unable-to-go-up-one-directory).
Hoping to address this later.

**Comment by radumas** [comment]
Yeah... the `data/` folder doesn't really work for the visualizations in sub-folders.
You could reference the absolute URL of the data. Just remember to update data independently of the code you're working on. e.g.: `https://cityoftoronto.github.io/bdit_king_pilot_dashboard/data/street_volumes.csv`

**Comment by q-schen** [comment]
Hm! The absolute URL works, I might stick with that.
